Dockerfile

```diff
@@ -35,32 +35,24 @@ WORKDIR /home/app/webapp
 RUN mkdir /etc/service/shoryuken
 COPY vendor/docker/shoryuken.sh /etc/service/shoryuken/run
 
-# Install Passenger monitor for Datadog
-# RUN wget https://github.com/Sjeanpierre/passenger-datadog-monitor/releases/download/v1.00/passenger-datadog-monitor && \
-#     mv passenger-datadog-monitor /usr/local/bin && \
-#     chmod +x /usr/local/bin/passenger-datadog-monitor && \
-#     mkdir /etc/service/passenger-datadog
-# COPY vendor/docker/passenger-datadog.sh /etc/service/passenger-datadog/run
-# RUN chmod +x /etc/service/passenger-datadog/run
+# Copy webapp folder
+COPY . /home/app/webapp/
+RUN mkdir -p tmp/pids && \
+    mkdir -p tmp/storage && \
+    chown -R app:app /home/app/webapp && \
+    chmod -R 755 /home/app/webapp
 
 # Install Ruby gems
-COPY Gemfile* /home/app/webapp/
+# COPY Gemfile* /home/app/webapp/
```

Comment:
Just remove this line, no need to leave commented.
What you've changed is that now you copy the entire webapp dir first to the env, which has the Gemfile in it.

I think previously it was trying to just install gems first before moving everything else.
This way round should work too